### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 --skip="./res/locales,./extlib"`

### CODE REVIEW

File: .appveyor/irc-notify.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def appveyor_vars():
                 break
 
             elif response[0] == 'PING':
-                file.send('PONG {}\r\n'.format(reponse[1]).encode('utf_8'))
+                file.send('PONG {}\r\n'.format(response[1]).encode('utf_8'))
```
Comment (luzpaz) [inline .appveyor/irc-notify.py]: Please double check this revision, thank you

**Comment by luzpaz** [comment]
CLA was already signed. See https://github.com/solvespace/solvespace/pull/304#issuecomment-404196056

**Comment by luzpaz** [comment]
:point_up_2:

**Comment by whitequark** [comment]
@luzpaz Sorry for the delay. Can you click the "recheck" link? Or does that not do anything?

**Comment by luzpaz** [comment]
Clicked 'recheck'. Not sure what to do now

**Comment by luzpaz** [comment]
I just manually re-signed the CLA. Recheck didn't work

**Comment by whitequark** [comment]
Thanks for the fixes!

**Pull request merged**